### ISSUE TITLE
How to use DiffAugment in Image to Image Translation?

### ISSUE THREAD

**Issue by hahahappyboy** [mit-han-lab/data-efficient-gans#93]
Hello, I try to use diffaugment in Image to Image Translation. The G use an label_image as input. The D use fake/real_image and label_image as input.
I don't know how to use DiffAugment to train them. Could you give me some advice? Thank You!
Image to Image Translation training example

```python
# update D
fake_img = G(label_img)
fake_score = D(fake_img.detach(),label_img)
real_score = D(real_img,label_img)
# Calculating D's loss based on real_scores and fake_scores...
...
....
# update G
fake_score = D(fake_img,label_img)
# Calculating G's loss based on fake_scores...
```

**Comment by zsyzzsoft** [comment]
You can try color and cutout augmentation on only real & fake images or translation augmentation on the concatenation of real (fake) and the label images.

**Comment by hahahappyboy** [comment]
OK, I will try it. Thank you！！！
